core/src/main/java/org/apache/accumulo/core/clientImpl/ServerClient.java

```diff
@@ -88,6 +88,9 @@ public static <T> T executeRaw(ClientContext context,
   public static <CT extends TServiceClient,RT> RT executeRaw(ClientContext context,
       TServiceClientFactory<CT> factory, ClientExecReturn<RT,CT> exec) throws Exception {
     while (true) {
+      if (Thread.currentThread().isInterrupted()) {
+        throw new AccumuloException("Thread interrupted");
```

Comment:
You could improve the `AccumuloException` by including more information and moving it down into the try/catch. You could include the `server` or there is a bunch of stuff in `ClientContext`.